image_segmentation/pytorch/runtime/trainer/trainer_factory.py

```diff
@@ -0,0 +1,15 @@
+from runtime.trainer.cuda_trainer import CUDATrainer
+from runtime.trainer.xla_trainer import XLATrainer
+
+
+def get_trainer(flags, model, train_loader, val_loader, loss_fn, score_fn, device, callbacks):
+    """Initializes and return the trainer (XLATrainer or CUDATrainer)"""
+    if flags.torch_xla:
```

Comment:
@thisisalbertliang 
Did you consider `flags.device == "xla"`?

Reply:
Thanks for the suggestion. I have replaced all occurrences of `flags.torch_xla` with `flags.device`.